SignalServiceKit/src/Messages/OWSBatchMessageProcessor.h

```diff
@@ -1,19 +1,19 @@
 //
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
 NS_ASSUME_NONNULL_BEGIN
 
 @class OWSSignalServiceProtosEnvelope;
-@class TSStorageManager;
+@class OWSStorage;
 
 // This class is used to write incoming (decrypted, unprocessed)
 // messages to a durable queue and then process them in batches,
 // in the order in which they were received.
 @interface OWSBatchMessageProcessor : NSObject
 
 + (instancetype)sharedInstance;
-+ (void)syncRegisterDatabaseExtension:(TSStorageManager *)storageManager;
++ (void)syncRegisterDatabaseExtension:(OWSStorage *)storage;
```

Comment:
We should be able to register our views on any "storage", e.g. both our primary and an "export" storage.